src/main/java/org/ow2/proactive/connector/iaas/rest/InstanceScriptRest.java

```diff
@@ -66,7 +66,7 @@ public Response executeScript(@PathParam("infrastructureId") String infrastructu
             @QueryParam("instanceId") String instanceId, @QueryParam("instanceTag") String instanceTag,
             final String instanceScriptJson) {
         log.info("Received request to execute script on infrastructure " + infrastructureId + " and instance id " +
```

Comment:
same as you did for instance id

Reply:
Leaving in small cap, just as other id in the logs